config/phpcs/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "varying-vagrant-vagrants/vvv",
+  "type": "metapackage",
+  "description": "An open source Vagrant configuration for developing with WordPress",
+  "keywords": [
+    "vagrant"
+  ],
+  "homepage": "https://varyingvagrantvagrants.org/",
+  "license": "MIT",
+  "support": {
+    "issues": "https://github.com/Varying-Vagrant-Vagrants/VVV/issues/"
+  },
+  "require": {
+    "dealerdirect/phpcodesniffer-composer-installer": "*",
+    "wp-coding-standards/wpcs": "*",
+    "automattic/vipwpcs": "*",
+    "phpcompatibility/php-compatibility": "*",
+    "phpcompatibility/phpcompatibility-paragonie": "*",
+    "phpcompatibility/phpcompatibility-wp": "*"
+  }
```

Comment:
we can simplify this down

https://github.com/Dealerdirect/phpcodesniffer-composer-installer/issues/134#issuecomment-798783941

```json
    "require": {
        "automattic/vipwpcs": "^2.2",
        "phpcompatibility/phpcompatibility-wp": "^2.1",
        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1"
    }
```

Ideally we want to stick to the major versions, I'd add in `wp-coding-standards/wpcs` restricted to the 2.x major version. If it's a choice between VIP coding standards and WP coding standards I'd prefer WP, but I also prefer no breakage if WPCS 3.x comes out and VIP are caught with their pants down (I'm not there to pull them back up)

Reply:
This is what I'm currently using on my personal projects:

```json
"require": {
    "automattic/vipwpcs": "*",
    "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1",
    "phpcompatibility/phpcompatibility-wp": "^2.1",
}
```

I left them at any because you weren't specifying in the old `require` statement.

Reply:
I wouldn't lock things down very much. Updating this regularly would be a pain, unless we have some sort of dependancy bot which recommends updates.

Reply:
According to the provision action here in GH, `"phpcompatibility/phpcompatibility-wp": "^2.1"` is not working.

Reply:
Fixed `phpcompatibility`. Should have been `2.1.1`.

Reply:
I disagree with not locking things down. The whole point of a platform like Vagrant is reproducibility through consistency. If you just install whatever the latest is, you're _asking_ for support issues that are hard to diagnose down the line.

Reply:
Also correct (I have two VVV-based Docker projects on my todo list... they're not fun!)

Reply:
It's expected that we install the latest, I'm deeply opposed to putting a lock file in

Reply:
Well you're the boss 😛